src/TroykaOLED.cpp

```diff
@@ -190,6 +184,12 @@ void TroykaOLED::print(String str, int x, int y) {
     _print(_codingCP866(data), x, y);
 }
 
+void TroykaOLED::print(char data, int x, int y) {
+    String s;
+    s += data;
+    print(s, x, y);
```

Comment:
Oooh-oh. Looks ineffective. Why not low-level?

```cpp
void TroykaOLED::print(char ch, int x, int y) {
    char buff[2] = { ch, 0x00 };
    _print(_codingCP866(buff), x, y);
}
```

Reply:
I checked your version. It's really more efficient than this one.